DQMOffline/Configuration/python/DQMOfflineHeavyIons_cff.py

```diff
@@ -15,6 +15,12 @@
 from DQM.BeamMonitor.AlcaBeamMonitorHeavyIons_cff import *
 from DQMOffline.L1Trigger.L1TriggerDqmOffline_cff import *
 
+### add for electron                                                                                                                                                                
+from DQMOffline.EGamma.electronOfflineClientSequence_cff import *
+from Validation.RecoEgamma.electronValidationSequence_cff import *
+from Validation.RecoEgamma.electronPostValidationSequence_cff import *
```

Comment:
ciao @davidlange6 
you are right. even if I don't see bounds to MC collections after the replacement, it would be nice to keep things separate and coherent.

@BetterWang can you move the sequence to ValidationHeavyIons_cff as suggested by david?
the DQM HI modules should be used for inclusion in the DQM/DQMOffline sequences.